### PR TITLE
[#2989] [#2990] Tailwind & VaAspectRatio docs pages, related bugfix

### PR DESCRIPTION
**Pull request by aluarius**
Closes: #2989 
Closes: #2990 

## Description
- [x] Tailwind docs page;
- [x] VaAspectRatio docs page;
- [x] VaAspectRatio added to @vuestic/nuxt components list;
- [x] Vuestic UI build type error fix.  

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: packages/ui/src/components/va-select/components/VaSelectContent/VaSelectContent.vue [hunk 1]
```diff
@@ -108,7 +108,7 @@ export default defineComponent({
   components: { VaBadge, VaIcon },
 
   props: {
-    value: { type: Array as PropType<SelectOption[]> },
+    value: { type: Array as PropType<SelectOption[]>, default: [] as SelectOption[] },
```
Comment (m0ksem) [inline packages/ui/src/components/va-select/components/VaSelectContent/VaSelectContent.vue:111]: Mustn't it be handled with useStateful composable? Even though, if it is internal component, I would better use `required: true` here.

File: packages/docs/page-config/ui-elements/aspect-ratio/index.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+export default definePageConfig({
+  blocks: [
+    block.title("aspectRatio.title"),
+    block.paragraph("aspectRatio.summaryText"),
+
+    block.subtitle("all.examples"),
+
+    block.headline("aspectRatio.examples.default.title"),
+    block.example("Default", { hideTitle: true }),
```
Comment (m0ksem) [inline packages/docs/page-config/ui-elements/aspect-ratio/index.ts:11]: Example block will render titles automatically, if not, you have a wrong structure in locales file.

```suggestion
    block.example("Default"),
```

File: packages/docs/page-config/ui-elements/aspect-ratio/index.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    block.headline("aspectRatio.examples.withOtherComponents.title"),
+    block.paragraph("aspectRatio.examples.withOtherComponents.text"),
+    block.example("WithOtherComponents", { hideTitle: true }),
```
Comment (m0ksem) [inline packages/docs/page-config/ui-elements/aspect-ratio/index.ts:15]:
```suggestion
    block.example("WithOtherComponents"),
```

File: packages/docs/page-config/styles/tailwind/code/configuringFull.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+// main.*
```
Comment (m0ksem) [inline packages/docs/page-config/styles/tailwind/code/configuringFull.ts:1]: Hm. We could get rid of these comments and provide fileName options for block.code.

**Review by m0ksem — CHANGES_REQUESTED** [review]
Minor issues

File: packages/docs/locales/en.json [hunk 5]
```diff
@@ -1572,6 +1573,7 @@
     "list": "List",
     "select": "Select",
     "parallax": "Parallax",
+    "tailwind": "Tailwind Support",
```
Comment (m0ksem) [inline packages/docs/locales/en.json:1576]:
```suggestion
    "tailwind": "Tailwind Integration",
```

**Review by m0ksem — CHANGES_REQUESTED** [review]

**Comment by m0ksem** [comment]
And I would reorder styles page:

- Colors
- CSS Variables
- Typography
- GrId (Make alert in grid and say that we recommend use Tailwind)
- Tailwind Integration
- Smart Helpers
- CSS reset (Make alert that we will remove it, because components work great without reset).

**Comment by aluarius** [comment]
> And I would reorder styles nav menu

Let's make it here https://github.com/epicmaxco/vuestic-ui/issues/2823

**Review by m0ksem — APPROVED** [review]

**Pull request merged**